### ISSUE TITLE
`SymGetLineFromAddr64` fails at runtime when compiling for 32bit

### ISSUE THREAD

**Issue by 0blu** [jeremy-rifkin/cpptrace#170]
When compiling for 32bit these two lines will fail at runtime:
https://github.com/jeremy-rifkin/cpptrace/blob/c7ff8cbf7924f21d61b94bf4443a8a548051fd94/src/symbols/symbols_with_dbghelp.cpp#L340-L341

There is a macro in `DbgHelp.h` (already included), which automaticity choses the right type.
So they just need to be replace with:
```cpp
IMAGEHLP_LINE line;
bool got_line = SymGetLineFromAddr(proc, addr, &displacement.b, &line);
```

**Comment by jeremy-rifkin** [comment]
Thanks for pointing this out, sounds reasonable to me. It's unfortunate the docs don't make this more clear :)

**Comment by jeremy-rifkin** [comment]
Should be resolved by 142e0b9ea48d2067d031a992a41a66042af6d207
